spyder/widgets/shortcutssummary.py

```diff
@@ -8,14 +8,15 @@
 
 
 # Standard library imports
+import sys
```

Comment:
This presumably wouldn't be needed if you eliminate the manual platform check and let Qt do it instead.